### DISCUSSION TITLE
Cannot export contents

### DISCUSSION THREAD

**Discussion by mnedoszytko** [Sequel-Ace/Sequel-Ace#1871] [Q&A] [answered]
Hi guys.
Am I missing something out? Since update to Ventura i cannot seem to find the option to export contents of table and thus do a full dump.
it all looks like this. before there were three columns on export dialog structure, drop, content. now only 2. 

<img width="734" alt="273124992-0f1dcb3d-65c2-41f1-96aa-26d8b4280fd8" src="https://github.com/Sequel-Ace/Sequel-Ace/assets/2605478/4960dbf4-a9c7-48ea-9e37-a3249288f613">

**Comment by samuelbradshaw** [comment, marked as answer]
I'm not sure what would cause it to get unchecked, but on the right there's a "Content" checkbox. After you turn that on, you should have three columns of checkboxes.

**Comment by mnedoszytko** [comment]
omg I'm so blind. It was there the whole time... Thank you.
